src/app/layouts/carousel/carousel-styling-sample/carousel-styling-sample.component.scss

```diff
@@ -0,0 +1,92 @@
+@import '~igniteui-angular/lib/core/styles/themes/index';
+
+/*BASIC THEMING*/
+
+$custom-theme: igx-carousel-theme(
+    $slide-background: rgb(247, 233, 233),
+    $button-background: gray,
+    $button-hover-background: #764949,
+    $border-radius: 10%,
+    $button-disabled-background:  rgb(184, 184, 184),
+    $button-disabled-arrow-color: gray,
+    $button-arrow-color: black,
+    $button-hover-arrow-color: #171717,
+    $indicator-dot-color: #515151,
+    $indicator-border-color: black,
+    $indicator-active-border-color: gray
+);
+
+/*ENHANCED APPROACHES*/
```

Comment:
Do we plan on including the commented out parts in the samples?

Reply:
@simeonoff  this is the way that all the styling samples are done. In all the other approaches are commented.